cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -83,6 +83,12 @@ public final class DataprocUtils {
   // This property needs to be overridden in cdap-site.
   public static final String TROUBLESHOOTING_DOCS_URL_DEFAULT = "";
 
+  // resources required by driver (default runtime job) running on driver pool node
```

Comment:
Let's not use "driver" term. Most often it's refered to Spark Driver, but in our case Spark driver runs in different place. In our case it's Runtime Job (io.cdap.cdap.runtime.spi.runtimejob.RuntimeJob) that will be running over there

Reply:
ack